lib/cli/src/commands/run.rs

```diff
@@ -848,6 +844,18 @@ pub(crate) fn try_run_package_or_file(
         return r.execute();
     }
 
+    // c:// might be parsed as a URL on Windows
+    let url_string = format!("{}", r.path.display());
+    if url_string.starts_with("http") {
+        if let Ok(url) = url::Url::parse(&url_string) {
```

Comment:
Would it be better to parse as a URL first, then check if `url.scheme() == "http" || url.scheme() == "https"`? 

I'm normally a bit dubious about "stringly-typed" APIs where you are doing things like `string.contains()` instead of parsing into a more strongly-typed representation like `Url` where you can properly inspect its properties.

For an example of what I mean, running `wasmer run http-server` would always error out saying something like *"unable to send a GET request to http-server"* because `"http-server".starts_with("http")` is `true`.

Reply:
done